spec/requests/preview/preview_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Preview::PreviewController, type: :request do
+  describe 'GET /preview' do
+    context 'with the preview environment is enabled' do
```

Comment:
Since the `SERVE_TEST_UI` `ENV` should be enabled in all tests in this context, we can simplify:

```suggestion
    context 'with the preview environment is enabled' do
      before do
        allow(ENV).to receive(:[]).with('SERVE_TEST_UI').and_return(true)
      end

```

Reply:
resolved in 02081f2